### PR TITLE
Artjom latest

### PR DESCRIPTION
**Pull request by artjomfrost**

### CODE REVIEW

File: front/src/components/BirthdayDialog.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Paper,
+    Table,
+    ThemeProvider } from "@mui/material";
+import React from "react";
+import globalTheme from "../styles/globalTheme";
+
+
+export default function BirthdayDialog() {
+
+    const data = [
```
Comment (klillenb) [inline front/src/components/BirthdayDialog.js:13]: why keep test data?

File: server/models/User.model.js [hunk 2]
```diff
@@ -94,8 +94,9 @@ userSchema.statics.login = async ({ username, password }) => {
 
 userSchema.statics.all = async () => {
     return new Promise(async (resolve, reject) => {
-        const users = await User.find({ deleted: false }, " -_id email fullname contact", {})
+        const users = await User.find({ deleted: false }, " -_id email fullname contact birthday", {})
         resolve(users)
+        console.log(users)
```
Comment (klillenb) [inline server/models/User.model.js:99]: oh no

**Review by klillenb — CHANGES_REQUESTED** [review]
minor fixes needed

**Review by RValdre — APPROVED** [review]
LGTM !

**Review by klillenb — APPROVED** [review]
omg

**Pull request merged**